build.gradle

```diff
@@ -58,7 +58,7 @@ allprojects {
     // with Navigation Native forced to be some snapshot version
     // if you need to use snapshots while development, just set `addSnapshotsRepo` to true manually
     def addSnapshotsRepo = project.hasProperty('ALLOW_SNAPSHOT_REPOSITORY') ? project.property('ALLOW_SNAPSHOT_REPOSITORY') : (System.getenv("ALLOW_SNAPSHOT_REPOSITORY")?.toBoolean() ?: false)
-    if (addSnapshotsRepo) {
+    if (true) {
```

Comment:
need revert

Reply:
Will be ready soon https://github.com/mapbox/mapbox-navigation-native/pull/5572#pullrequestreview-918657453